src/docs/sphinx/advancedExamples/validationStudies/Index.rst

```diff
@@ -45,6 +45,8 @@ Validation and Verification Studies
  
    proppantSlotTest/Example
 
+   faultVerification/Example
```

Comment:
We desperately need a re-org of our examples, perhaps adding another directory level.  Right now it is a mish-mash.  Perhaps we can use this PR as an opportunity?

Reply:
Agree, let's discuss it and work on a separate PR for reorganizing examples